src/main/external-resources/renderers/Yamaha-RXA2050.conf

```diff
@@ -0,0 +1,39 @@
+#-------------------------------------------------------------------------------------------
+# Profile for Yamaha RX-A2050.
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Yamaha RX-A2050
+RendererIcon = yamaharxa1010.png
+
+# =============================================
+# This renderer has sent the following string/s:
+#
+# friendlyName=RX-A2050 D474AF
+# udn=uuid:9ab0c000-f668-11de-9976-00a0ded474af
+# manufacturer=Yamaha Corporation
+# modelName=RX-A2050
+# modelNumber=A2050
+# modelDescription=AV Receiver
+# manufacturerURL=http://www.yamaha.com/
+# modelURL=http://www.yamaha.com/
+# =============================================
+# http://usa.yamaha.com/products/audio-visual/aventage/rx-a2050_black_u/
+
+UpnpDetailsSearch = RX-A2050
+
+Video = false
+Image = false
+SeekByTime = true
+TranscodeAudio = WAV
+StreamExtensions = dff,dsf,l16,lpcm

```

Comment:
@valib Are you sure about that ?
Because when i looked at the code, as i reported in a commit, i didn't understood that.
https://github.com/UniversalMediaServer/UniversalMediaServer/blob/978bd1584cbce21989bb8c89732442ae81355d91/src/main/java/net/pms/configuration/RendererConfiguration.java#L2210-L2215